### ISSUE TITLE
[FEATURE] Export RBM model

### ISSUE THREAD

**Issue by miguelgfierro** [microsoft/recommenders#1510]
### Description

Problem when saving RBM model, following the Wide&Deep model implementation https://nbviewer.jupyter.org/github/microsoft/recommenders/blob/staging/examples/00_quick_start/wide_deep_movielens.ipynb#4.3-Export-Model but when I exported the RBM model I received the following error

`AttributeError: 'RBM' object has no attribute 'experimental_export_all_saved_models'`

### In which platform does it happen?

Linux VM

### How do we replicate the issue?

```
train_fn = tf_utils.pandas_input_fn(
    df=train_df,
    y_col='rating',
    batch_size=100,
    num_epochs=None,  .
    shuffle=True,
    seed=42,
)
# Unique users in the dataset
user_ids = tf.feature_column.categorical_column_with_vocabulary_list("userID", user["userID"].values)
item_ids = tf.feature_column.categorical_column_with_vocabulary_list("movieID", item["movieID"].values)

fn = [
        user_ids,
        item_ids,
        tf.feature_column.crossed_column(
                [user_ids, item_ids], hash_bucket_size=1000
        ),
    ]

exported_path = tf_utils.export_model(
    model=model,
    train_input_fn=train_fn,
    eval_input_fn=tf_utils.pandas_input_fn(
        df=test_df, y_col='rating'
    ),
    tf_feat_cols= fn
    base_dir=EXPORT_DIR_BASE,
)
```

### Other Comments
related to #878

**Comment by miguelgfierro** [comment]
FYI @pradnyeshjoshi

**Comment by miguelgfierro** [comment]
fixed in #1618
